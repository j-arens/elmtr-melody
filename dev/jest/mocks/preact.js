module.exports = {
    unmountComponent: jest.fn(),
};
