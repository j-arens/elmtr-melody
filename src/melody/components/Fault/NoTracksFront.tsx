/* tslint:disable:max-line-length */
import { cySelector } from '@utils/index';
import { h } from 'preact';
const s = require('./style.scss');

export default () => (
    <p class={s.ErrorMsg} {...cySelector('error-msg')}>
        No tracks are loaded. If you think this an error try refreshing the page, otherwise please contact the site administrator.
    </p>
);
