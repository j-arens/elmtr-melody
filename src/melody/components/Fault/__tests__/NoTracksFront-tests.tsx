import { render } from 'preact-render-to-string';
import { h } from 'preact';
import NoTracksFront from '../NoTracksFront';

describe('NoTracksFront', () => {
    it('renders', () => {
        const component = render(<NoTracksFront/>);
        expect(component).toMatchSnapshot();
    });
});
