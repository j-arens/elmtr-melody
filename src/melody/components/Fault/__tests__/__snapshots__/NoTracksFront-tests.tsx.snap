// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoTracksFront renders 1`] = `
<p class="ErrorMsg" data-cy="error-msg">No tracks are loaded. If you think this an error try refreshing the page,
  otherwise please contact support.</p>
`;
